node/src/accountant/receivable_dao.rs

```diff
@@ -360,34 +294,115 @@ impl ReceivableDaoReal {
         }
     }
 
-    fn row_to_account(row: &Row) -> rusqlite::Result<ReceivableAccount> {
-        let wallet: Result<Wallet, rusqlite::Error> = row.get(0);
-        let balance_result = row.get(1);
-        let last_received_timestamp_result = row.get(2);
-        match (wallet, balance_result, last_received_timestamp_result) {
-            (Ok(wallet), Ok(balance), Ok(last_received_timestamp)) => Ok(ReceivableAccount {
-                wallet,
-                balance,
-                last_received_timestamp: dao_utils::from_time_t(last_received_timestamp),
-            }),
-            tuple => panic!("row_to_account(): Database is corrupt: RECEIVABLE table columns and/or types: {:?}", tuple),
+    fn form_receivable_account(row: &Row) -> rusqlite::Result<ReceivableAccount> {
```

Comment:
I guess I'd rather see "make" or "create" or "generate" than "form." I've been looking at "form" for a couple of days now and it still looks like a misspelling of "from."